packages/angular_devkit/core/node/scheduler.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { jobs, json, schema } from '../src';
+import { ModuleNotFoundException, resolve } from './resolve';
+
+export class NodeModuleJobScheduler extends jobs.SimpleScheduler {
```

Comment:
I don't want to mix registration and scheduling. It could be two distinct interfaces, yes.

The subclassing was because the code is fairly similar (and complex) and it doesn't make sense to copy paste it. You don't have to subclass the scheduler; I could see a server that schedule jobs through HTTP requests, and have a client scheduler that only forward `has()`, `schedule()` and `getDescription`.

Reply:
Done, two different interfaces and classes.